styles.css

```diff
@@ -8,6 +8,22 @@ td.yX { /*spacing out the table rows*/
   padding-bottom: 20px !important;
 }
 
+div.pH.a9q {
+	display: none;
+}
+
+span.aXw.T-KT {
+	display: none;
+}
+
+div.oZ-jc.T-Jo.J-J5-Ji {
+	display: none;
+}
+
+div.pH-A7.a9q {
+	display: none;
+}
+
```

Comment:
Since the classnames are non-descriptive, it might be useful to add comments to these rules to describe exactly which element you're hiding.